apps/throttles.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework.throttling import SimpleRateThrottle
+from influxdb_metrics.loader import log_metric
+
+
+class OrgApiLimit(SimpleRateThrottle):
+    """
+    API rate limiter for req/day
+    """
+    scope = 'org_api_limit'
+    rate = '1000/day'  # default rate
+
+    def get_cache_key(self, request, view):
+        # Verify user belongs to billable organization
+        if request.user.is_authenticated and 'payload' in request.user.token \
+                and 'organization_id' in request.user.token.payload:
+            self.rate = self.request.user.token.get('org_api_limit', None)
+
+            if self.rate:
+                self.rate = f'{self.rate}'
+
+                ident = request.user.payload.organization_id
```

Comment:
Either reuse this local var in the log_metric call, or get rid of it and reference the token directly in the return statement.

Reply:
SGTM